### PR TITLE
MAINT: organize and clean up files

### PR DESCRIPTION
**Pull request by redeboer**
This PR cleans up the main directory. The idea is to only have developer configs in there, such as `pixi.toml` and `.pre-commit-config.yaml`. The rest is organised in folders.

### CODE REVIEW

**Comment by redeboer** [comment]
@mmikhasenko are these files still required:
- [`HS3_base.json`](https://github.com/RUB-EP1/amplitude-serialization/blob/ecee71af8f77bc0360b3c5d8bc85592802652eb0/models/HS3_base.json)
- [`HS3_extended.json`](https://github.com/RUB-EP1/amplitude-serialization/blob/ecee71af8f77bc0360b3c5d8bc85592802652eb0/models/HS3_extended.json)

If so, is there a plan to document them or integrate them into CI?

Similar for the two Julia scripts that were moved to `docs/julia`.

**Comment by mmikhasenko** [comment]
The files with HS3 are not needed. I'm deleting them moving notes to md.

**Comment by redeboer** [comment]
> I'm deleting them moving notes to md.

If you want them on the page on HS3 compatibility, I'm fine with keeping them and including them through Markdown like so:

https://github.com/RUB-EP1/amplitude-serialization/blob/ecee71af8f77bc0360b3c5d8bc85592802652eb0/docs/index.md#L149-L151

That has the benefit that you can still validate them (see e.g. #30).

File: docs/HS3.md [hunk 1]
```diff
@@ -20,3 +20,107 @@ In parallel to development of the model description format, compatibility with H
...
+"analyses": [
+  {
+    "name": "my_ana",
+    "parameter_domain": "default",
+    "prior": "my_prior_dist"
+  }
+]
+```
+
+**Analyses** specifies how kinematic variables are distributed within the phase space, using priors (here, `"prior": "my_prior_dist"`) to influence their sampling distributions.
```
Comment (redeboer) [inline docs/HS3.md:106]: Is that related to this proposal?
https://github.com/hep-statistics-serialization-standard/hep-fit-serialization/blob/cef8186/proposals/draft/hs3_example_file_high_level.pseudo_json#L44-L47

File: docs/julia/Manifest.toml
[the remark this answers is not in the document]
Comment (redeboer) [reply docs/julia/Manifest.toml:1]: Ah nvm, I see you have defined a package name in `Project.toml`.

File: docs/julia/Project.toml [hunk 2]
```diff
@@ -1,3 +1,7 @@
+name = "AmplitudeSerialization"
+uuid = "1fc14086-b083-4993-a519-4e0daf556ad7"
+version = "0.1.0"
+
```
Comment (redeboer) [inline docs/julia/Project.toml:4]: See https://github.com/RUB-EP1/amplitude-serialization/pull/29/files#r1598516692

For my understanding, what is the benefit of defining a project name (and version). Isn't this only required for libraries?

File: docs/HS3.md (hunk 1 above)
Comment (mmikhasenko) [reply docs/HS3.md:106]: I am not sure, but I doubt that that is from the proposal: there should be a standard way of sampling with priors
The way how simulation works with HS3 was explained to me by Carsten.

**Comment by mmikhasenko** [comment]
was a revert on a wrong branch first. Now it's here.
